extensions/AzureCosmosDBMongoDB/AzureCosmosDBMongoDB/AzureCosmosDBVectorSearchType.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+// ReSharper disable InconsistentNaming
+
+namespace Microsoft.KernelMemory.MemoryDb.AzureCosmosDBMongoDB;
+
+public enum AzureCosmosDBVectorSearchType
+{
+    vector_ivf,
+    vector_hnsw
```

Comment:
please rename these to follow the naming convention, something like `VectorIVF` and `VectorHNSW` would be better. If different values are needed for serialization, you can use the serialization attributes. Thanks

Reply:
Updated the values.